unit-tests/shared/src/test/scala/org/scalanative/testsuite/javalib/net/SocketTest.scala

```diff
@@ -274,7 +274,7 @@ class SocketTest {
     val s6 = new Socket()
     try {
       assertThrows(
-        "a5",
+        "a6",
```

Comment:
Here I'm bit by technical dept.  When there is one
assertion in  test, it is easy to tell which one failed.

When there is more than one, figuring out which one failed
is a pain.

In the very old days, assertMumble() did not take a message
argument and _lots_ of code was written that way.

When tearing my hair out trying to debug errors in this file,
I quickly add messages with just information to distinguish them, and give me some idea of where in the test (early, mid, late), the test failed.  In a better life, I would have added useful, descriptive text.

Here, "a5" indicates the fifth asserting in the Test, not that anybody but me would understand the undocumented & cryptic convention.

"a6" is just as distinguishable and matches the variable name.

I advise _against_ doing a sweep through the file and changing other points, unless it is to give them useful descriptive names.